Dangerfile

```diff
@@ -0,0 +1,24 @@
+def version
+  lib = File.expand_path("../lib", __FILE__)
+  $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+  require 'fastlane/plugin/wpmreleasetoolkit/version'
+  Fastlane::Wpmreleasetoolkit::VERSION
+end
+
+def gemfile_lock_version
+  gemfile_lock = File.read('./Gemfile.lock')
+  gemfile_lock.scan(/fastlane-plugin-wpmreleasetoolkit \((\d+.\d+.\d+)\)/).last.first
+end
+
+# Before checking the version, get rid of any change that `bundle install`
+# might have done.
+`git checkout Gemfile.lock &> /dev/null`
+
+if version.to_s != gemfile_lock_version.to_s
+  message = %Q{
+The version in the `Gemfile.lock` (`#{gemfile_lock_version}`) doesn't match the one in `version.rb` (`#{version}`).
```

Comment:
I realize it's a bit wasteful to have those functions run twice (if the numbers don't match), but in the context of this file, I think the better readability trumps the runtime duplication.